entrypoint.sh

```diff
@@ -35,5 +37,10 @@ do
     sleep 2
 done
 
+while [ ! -e $c ]
```

Comment:
Within the limits of how much bash complexity we're willing to tolerate, do you think this might be less error prone:

```
while [ ! -e $a ] || [ ! -e $b ] || [ ! -e $c ]; do
  sleep 2
done
```

alternatively given we copy things in one-after-another we could just wait on the last thing, or as a 3rd option have an explicit "go signal" system like we use elsewhere?